lib/keyboard_layouts.dart

```diff
@@ -115,6 +115,36 @@ const graphite = KeyboardLayout(
   ],
 );
 
+const galliumV2 = KeyboardLayout(
+  name: 'Gallium V2',
+  keys: [
```

Comment:
Many keys misplaced. Refer to the [Gallium repo](https://github.com/GalileoBlues/Gallium) on implementation of ANSI Gallium V2/Gallium_rowstag. Attached is a picture of MonkeyType's implementation of Gallium V2.
![image](https://github.com/user-attachments/assets/d2b63437-9971-4a6c-8d90-44664acf43cd)